cmd/osbuild-service-maintenance/db.go

```diff
@@ -10,37 +10,34 @@ import (
 )
 
 func DBCleanup(dbURL string, dryRun bool, cutoff time.Time) error {
-	archs := []string{"x86_64"}
-	jobType := "osbuild"
-
 	jobs, err := dbjobqueue.New(dbURL)
 	if err != nil {
 		return err
 	}
 
-	var jobTypes []string
-	for _, a := range archs {
-		jobTypes = append(jobTypes, fmt.Sprintf("%s:%s", jobType, a))
-	}
-
-	jobsByType, err := jobs.JobsUptoByType(jobTypes, cutoff)
+	// The results of these jobs take up the most space and can contain sensitive data. Delete
```

Comment:
True, but the result data might be useful long term and shouldn't contain sensitive info like activation keys. There's also business metrics things coming up where we might need to know if a job succeeded or not.

To conserve space on stage we should just delete them (after #2712) imo, but we can also do that via scheduled queries in app-intf rather than the maintenance, could also add it here of course.

Reply:
Sounds good, thanks for explaining.